docs/guide/dev/integration/react/README.md

```diff
@@ -215,6 +215,60 @@ component.editor.getData();
 
 Please note that this property is initialised asynchronously, after mounting the component.
 
+## Properties
+
+CKEditor4 React integration supports properties:
+- [config](#config) : `Object`
+- [data](#data) : `''`
+- [name](#name) : `String`
+- [readOnly](#readonly) : `Boolean`
+- [style](#style) : `Object`
+- [type](#type) : `'classic'` | `'inline'`
+- [onBeforeLoad](#onbeforeload) : `Function`
+- [onNamespaceLoaded](#onnamespaceloaded) : `Function`
+
+### config
+
+**Default**: `{}`
+
+Stores configuration settings. All available configuration options can be found in the API documentation. Refer to the {@linkapi CKEDITOR.config config} object definition. This config will be applied to editor instances separately.
+
+### data
+
+**Default**: `''`
+
+Data will be used to initially fill editor content. It's passed to {@linkapi CKEDITOR.editor#setData} but without any other arguments.
+
+### name
+
+Provided name became editor HTML element `id` and `name` properties.
+
+### readOnly
+
+**Default**: `false`
+
+Whenever editor should be read-only. Overrides {@linkapi CKEDITOR.config#readOnly config readOnly} before it's used to initialize editor instance.
```

Comment:
I mean hear to describe this behaviour:
https://github.com/ckeditor/ckeditor4-react/blob/4a4159dbd03bb10c206888a748bd261842e5c510/src/ckeditor.jsx#L55-L56

So no matter what config property says about `readOnly` - it's overridden, and then config is used to initialize editor.

Or you mean I described it in a not understandable manner?

Reply:
It might sound not clear enough. I decided to reorganize the entire doc. I moved the already existing section about `readOnly` into property description with some adjustment.

I also moved editor `type` and `config` into properties. There were already provided samples and more proper descriptions.